script.js

```diff
@@ -1,11 +1,20 @@
 // Create an array called `colors` that contains three colors.
-
+var colors = ["red", "white", "blue"]{
+}
 // Log the last item in the array to the console.
-
+console.log
```

Comment:
this is referencing the `console.log` method, we need to invoke and provide the last array item as an argument, so:

`console.log(colors[colors.length - 1]);`